### PR TITLE
Remove Itenerary Model and Updated generateTrip Input Schema

### PR DESCRIPTION
**Pull request by L0Lmaker**
#113 Removing Itenerary Model

### CODE REVIEW

File: server/openai/openaiClient.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ let openai
 const configuration = new Configuration({
   apiKey: process.env.OPEN_AI_API_KEY || 'add-alt-key-here',
```
Comment (AndyLiang1) [inline server/openai/openaiClient.js]: Hiya could you put this in `config.js`, the one in the folder called `config`?

File: server/openai/test.js [hunk 2]
```diff
@@ -3,31 +3,22 @@
  Run via `node openai/test.js` from `server/` or wherever the .env file with the openai key is
 */
 
-const generateItinerary = require('./generateItenerary')
-const Itinerary = require('../models/itineraryModel')
+const generateItinerary = require('./generateTrip')
+const TripModel = require('../models/TripModel')
 
-const exampleItinerary = new Itinerary({
+const exampleItinerary = new TripModel({
```
Comment (AndyLiang1) [inline server/openai/test.js]: Rename, let's not use the word Itinerary

File: server/openai/test.js [hunk 3]
```diff
@@ -3,31 +3,22 @@
  Run via `node openai/test.js` from `server/` or wherever the .env file with the openai key is
 */
 
-const generateItinerary = require('./generateItenerary')
-const Itinerary = require('../models/itineraryModel')
+const generateItinerary = require('./generateTrip')
```
Comment (AndyLiang1) [inline server/openai/test.js]: Rename, let's not use the word Itinerary :)

File: server/openai/test.js [hunk 4]
```diff
@@ -3,31 +3,22 @@
...
-      Preferences: ${exampleItinerary.preferences.join(', ')}
-    `
-
-    const itinerary = await generateItinerary(itineraryConstraints)
+    const itinerary = await generateItinerary(exampleItinerary)
 
     console.log(`Generated itinerary for given constraints:`)
-    console.log(itineraryConstraints)
+    console.log(exampleItinerary)
     console.log(itinerary)
```
Comment (AndyLiang1) [inline server/openai/test.js]: Remove console logs pls :)

File: server/openai/test.js [hunk 5]
```diff
@@ -3,31 +3,22 @@
...
-      Budget: $${exampleItinerary.budget}
-      Days: ${exampleItinerary.days}
-      Stages per day: ${exampleItinerary.stages}
-      Preferences: ${exampleItinerary.preferences.join(', ')}
-    `
-
-    const itinerary = await generateItinerary(itineraryConstraints)
+    const itinerary = await generateItinerary(exampleItinerary)
 
     console.log(`Generated itinerary for given constraints:`)
```
Comment (AndyLiang1) [inline server/openai/test.js]: Remove console logs pls :)

Comment (L0Lmaker) [reply server/openai/test.js]: this is for the test script. Without the console logs it is useless. There are no log statements in the `generateTrip.js` file which is what will be used by BE

File: server/openai/openaiClient.js (hunk 1 above)
Comment (L0Lmaker) [reply server/openai/openaiClient.js]: will do

File: server/openai/test.js (hunk 4 above)
Comment (L0Lmaker) [reply server/openai/test.js]: https://github.com/cpsc455-bugstorm/TravelersTea/pull/132#discussion_r1253558909

File: server/openai/test.js (hunk 5 above)
Comment (AndyLiang1) [reply server/openai/test.js]: Sure. Resolving...

**Review by AndyLiang1 — APPROVED** [review]

**Pull request merged**